### DISCUSSION TITLE
Won't build! Using the "Building from Source" method.

### DISCUSSION THREAD

**Discussion by p1nkr0b0t** [fosskers/aura#814] [Q&A]
Pacman (when in the `aura-bin` directory), couldn't find `aura-bin`, the package that `MAKEPKG` made! So, I installed the Haskell Stack and installed it that way with the following, output is in a Pastebin link.

https://pastebin.com/8xAH5VWP

**Comment by fosskers** [comment]
Ah I see, I believe the docs are out of date. If instead you entered the `haskell/` folder first, then ran `stack install`, it should work. Can you confirm this?

**Reply by p1nkr0b0t** [reply]
Hm? Where would `haskell/` be? (This might seem like a rather basic question, since I am still a _liiiittle_ new to Linux, but I'm limited to `pacman` at the moment as I do not have an AUR helper.)
Can you give me an example as to what that would look like as a `command`?

**Reply by fosskers** [reply]
Gotcha. I can see from your log that you've cloned the Aura repository with `git` and then ran `stack install` from within the project root. So you did:
```
git clone ...
cd aura
stack install -- aura
```
but before the `stack` command, just enter the `haskell/` subdirectory first:
```
cd haskell
```
and then run `stack install -- aura`. Let me know if it works.

**Reply by p1nkr0b0t** [reply]
Alright, thank you. I will try it when I get home.

**Comment by p1nkr0b0t** [comment]
@fosskers Hey, it's seemed to have worked! :D
The sheer amount of lines is more than my terminal can scroll lmao.
Though, on the very last one, I see this:
`Warning: Installation path /home/azarel/.local/bin not found on the PATH environment variable.`
Should I be worried about that, or just leave it alone?

**Reply by p1nkr0b0t** [reply]
Nvm, tried installing a random package, `hollywood` for the cool hackerman effects, and...
`[azarel@ArchLinux haskell]$ aura -A hollywood`
`bash: aura: command not found`
:(

**Reply by fosskers** [reply]
Right! You'll need to add `/home/azarel/.local/bin` to your `PATH`. Since your shell is probably `bash`, you can do a search for "how to add directory to PATH on Linux" and there should be plenty of examples of how to do that.

Let me know how it goes!
